modules/recommended-configurable-storage-technology.adoc

```diff
@@ -60,21 +60,22 @@ storage would be an anti-pattern. One volume per elasticsearch is required.
 
 [NOTE]
 ====
-A scaled registry is an {product-title} registry where three or more pod replicas are running.
+A scaled registry is an {product-title} registry where two or more pod replicas are running.
```

Comment:
s/pod/Pod

I know this isn't part of the direct changes, but figured I'd flag it. :slightly_smiling_face: